projects/plugins/jetpack/modules/widgets/contact-info.php

```diff
@@ -163,7 +163,7 @@ public function widget( $args, $instance ) {
 				}
 			}
 
-			if ( is_email( trim( $instance['email'] ) ) ) {
+			if ( is_email( trim( $instance['email'] ?? '' ) ) ) {
```

Comment:
What if we were to bail even before to make the email check? Maybe it would help with performance a bit?

```suggestion
			if (
				! empty( $instance['email'] )
				&& is_email( trim( $instance['email'] ) )
			) {
```

Reply:
We can't bail early because there's additional logic that needs to run (i.e. outputting `$instance['hours']` and the closing `</div>`).

Reply:
Ah, just looked at your example, and you didn't mean "return". :^)